kahuna/public/js/components/gr-panel-button/gr-panel-button-small.html

```diff
@@ -11,9 +11,10 @@
             gr-track-click-data="ctrl.trackingData('Hide panel')"
             aria-label="Hide {{ctrl.name}} panel">
         <gr-icon>{{ctrl.icon}}</gr-icon>
```

Comment:
```suggestion
        <gr-icon style="margin-bottom: 2.5px;">{{ctrl.icon}}</gr-icon>
```
minor thing but I think this looks better...
| Before | After |
| --- | --- |
| ![image](https://github.com/guardian/grid/assets/19289579/b11c9510-f49c-4887-93ec-e60e2b8586eb) | ![image](https://github.com/guardian/grid/assets/19289579/b39841db-9623-48d9-8609-496f8853ce3e) |